<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalc</class>
 <widget class="QMainWindow" name="DepositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(153, 193, 241)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Годовая</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>размещения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_withdraw">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton_withdraw {
     background-color: #32629D;
 }</string>
    </property>
    <property name="text">
     <string>Снятие</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>проценты</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>147</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Статус</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>начала срока</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Капитализация</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_capital">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Ежемесячно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежеквартально</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежегодно</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_term">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1 месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 месяца</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 год</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 лет</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>610</width>
      <height>501</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Выплаченные проценты</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма вклада</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Налоговая ставка</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_payment_withdrawal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2100</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2023</year>
      <month>8</month>
      <day>24</day>
     </datetime>
    </property>
    <property name="date">
     <date>
      <year>2023</year>
      <month>8</month>
      <day>24</day>
     </date>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_tax">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма на вкладе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_count">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton_count {
     background-color: #ffa000;
 }</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>пополнения/снятия</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выплаты</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_pw_amount">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_payment">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Ежемесячно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежеквартально</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежегодно</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>580</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton_clear {
     background-color: #7279D8;
 }</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_deposit_amount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>10000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма налога</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_deposit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>70</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTextEdit#textEdit_deposit {
     background-color: rgb(255, 255, 255);
	border-radius: 8px
 }</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_start">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>140</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2100</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2023</year>
      <month>8</month>
      <day>24</day>
     </datetime>
    </property>
    <property name="date">
     <date>
      <year>2023</year>
      <month>8</month>
      <day>24</day>
     </date>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_rate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>вклада</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_tax">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTextEdit#textEdit_tax {
     background-color: rgb(255, 255, 255);
	border-radius: 8px
 }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>-30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_percentages">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTextEdit#textEdit_percentages {
     background-color: rgb(255, 255, 255);
	border-radius: 8px
 }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_top">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton_top {
     background-color: #32629D;
 }</string>
    </property>
    <property name="text">
     <string>Пополнение</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>21</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>пополнения/снятия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cрок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cумма</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начисленные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>процентная ставка %</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1328</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDepositCalc">
    <property name="title">
     <string>DepositCalc</string>
    </property>
    <addaction name="actionCreditCalc"/>
    <addaction name="actionSmartCalc"/>
   </widget>
   <addaction name="menuDepositCalc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreditCalc">
   <property name="text">
    <string>CreditCalc</string>
   </property>
  </action>
  <action name="actionSmartCalc">
   <property name="text">
    <string>SmartCalc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
